.asf.yaml

```diff
@@ -28,7 +43,22 @@ github:
     rebase:  true
 
   protected_branches:
-    main: { }
+    main:
+      required_status_checks:
+        # strict means "Require branches to be up to date before merging".
+        strict: false
+        # contexts are the names of checks that must pass
+        contexts:
+          - Scalafmt
```

Comment:
As per https://github.com/apache/incubator-pekko-sbt-paradox/pull/7, this needs to be changed to `- Code is formatted` rather than `- Scalafmt`